### ISSUE TITLE
User import hashing or generating password. Ideas question ?

### ISSUE THREAD

**Issue by kerbs-ns** [simonhamp/laravel-nova-csv-import#65]
If i need to import users

nova user resource
```
  Password::make('Password')
                ->onlyOnForms()
                ->creationRules('required', Rules\Password::defaults())
                ->updateRules('nullable', Rules\Password::defaults()),
```

how one what is strategy to import users with passwords and hash them or generate some random password if not part of import ?

**Comment by simonhamp** [comment]
@kerbs-ns I've just seen your PR (#67) for this. Thanks! I will review this soon

**Comment by simonhamp** [comment]
Finished in #74 and released in [v0.7.0](https://github.com/simonhamp/laravel-nova-csv-import/releases/tag/v0.7.0)
